src/index.js

```diff
@@ -0,0 +1,75 @@
+import './styles/sass/main.sass';
+import './assets/refresh.png';
+import './assets/enter.png';
+import './assets/trash-can.png';
```

Comment:
Nice organisation of code. Please have you tried using **import { refresh.png, enter.png, trash-can.png } from './assets'**?

Reply:
this one didnt work for me

Reply:
I tried `import * from './assets'` but didnt work either